pkg/internal/test/starts_with_matcher.go

```diff
@@ -0,0 +1,54 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/types"
+)
+
+// StartWith is a GomegaMatcher verifying if given string starts with prefix
+func StartWith(expected string) types.GomegaMatcher {
```

Comment:
How about `StartsWith` in the 3rd person form - wouldn't that sound more natural?

Reply:
I asked myself the very same question, but that doesn't sound proper English to me, "Expect something to start with" is what we should be reading here.